File: app/src/main/java/com/theone/simpleshare/ui/paired/PairedViewModel.java
package com.theone.simpleshare.ui.paired;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

import java.util.ArrayList;

public class PairedViewModel extends ViewModel {

    private final MutableLiveData<String> mText;
    private final MutableLiveData<ArrayList<PairedItem>> mItemList;
    public PairedViewModel() {
        mText = new MutableLiveData<>();
        mText.setValue("This is home fragment");

        mItemList = new MutableLiveData<>();
        mItemList.setValue( new ArrayList<>());
    }

    public LiveData<String> getText() {
        return mText;
    }

    public LiveData<ArrayList<PairedItem>> getList(){
        return mItemList;
    }

}

File: app/src/main/kotlin/com/theone/simpleshare/ui/paired/PairedViewModel.kt
package com.theone.simpleshare.ui.paired

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel
import java.util.ArrayList

class PairedViewModel : ViewModel() {
    private val mText: MutableLiveData<String>
    private val mItemList: MutableLiveData<ArrayList<PairedItem>>
    val text: LiveData<String>
        get() = mText
    val list: LiveData<ArrayList<PairedItem>>
        get() = mItemList

    init {
        mText = MutableLiveData()
        mText.value = "This is home fragment"
        mItemList = MutableLiveData()
        mItemList.value = ArrayList<PairedItem>()
    }
}